printer_mods/prasadp4009/SW_SB_CRTouch/README.md

```diff
@@ -0,0 +1,19 @@
+# Voron SwitchWire StealthBurner CR Touch Mod
```

Comment:
Switchwire should be written without a capital W.  just `Switchwire`

Reply:
Fixed it.